docs/user-guide/validate-files.rst

```diff
@@ -65,3 +66,53 @@ To use these functions it is necessary to create a `Validator` object and initia
                                    stories='data/stories.md')
 
   validator.verify_all()
+
+Test Story Files for Conflicts
+------------------------------
+
+In addition to the default tests described above, you can also do a more in-depth structural test of your stories.
+In particular, you can test if your stories are inconsistent, i.e. if different bot actions follow from the same dialogue history.
+If this is not the case, then Rasa cannot learn the correct behaviour.
+
+Take, for example, the following two stories:
+
+.. code-block:: md
+
+  ## Story 1
+  * greet
+    - utter_greet
+  * inform_happy
+    - utter_happy
+    - utter_goodbye
+
+  ## Story 2
+  * greet
+    - utter_greet
+  * inform_happy
+    - utter_goodbye
+
+These two stories are inconsistent, because Rasa cannot know if it should predict ``utter_happy`` or ``utter_goodbye`` 
```

Comment:
Wondering if this might be better

```suggestion
These two stories are inconsistent, because Rasa doesn't know if it should predict ``utter_happy`` or ``utter_goodbye``
```